### PR TITLE
[VLM] Report multi_modal_placeholders in output

### PR DESCRIPTION
**Pull request by lk-chen**
Report multi_modal_placeholders in output

This helps inspect VLM model performance, e.g. distinguish text tokens and multi-modal embeddings.

related: #8346

### CODE REVIEW

**Review by DarkLight1337 — APPROVED** [review]
Thanks for improving the developer experience!

**Comment by DarkLight1337** [comment]
Do you get these test errors locally?

**Comment by DarkLight1337** [comment]
Please fix the lint errors.

**Comment by lk-chen** [comment]
> Please fix the lint errors.

Done. Could you help me remove the accidentally-added reviewers? Those were added when I tried to fix DCo

File: tests/models/decoder_only/vision_language/test_pixtral.py [hunk 1]
```diff
@@ -23,6 +28,8 @@
 if TYPE_CHECKING:
     from _typeshed import StrPath
 
+logger = init_logger(__name__)
```
Comment (DarkLight1337) [inline tests/models/decoder_only/vision_language/test_pixtral.py:31]: Can we remove this logger?

**Review by comaniac — APPROVED** [review]
LGTM. The reviewers were added automatically due to code owners so it's fine.

Also cc @ywang96 @mgoin

**Pull request merged**